docs/user-guide/concepts.md

```diff
@@ -13,6 +13,7 @@ namespaces are, and _why_ they behave the way they do.
 * [Basic concepts](#basic)
   * [Parents, children, trees and forests](#basic-trees)
   * [Full namespaces and subnamespaces](#basic-subns)
+  * [Hierarchical Resource Quotas](#hierarchical-resource-quota)
```

Comment:
Use lowercase to match other headings, but you can add the acronym. Also, I'd use the acronym as the anchor, e.g.:

`[Hierarchical resource quotas (HRQs)](#hrq)`

Reply:
thanks!
you are a pro!